### DISCUSSION TITLE
Command to watch a directory?

### DISCUSSION THREAD

**Discussion by rlovell3** [bensadeh/tailspin#151] [Q&A] [answered]
What is the tspin command to watch the directory /var/log?

**Comment by bensadeh** [comment]
I believe `tspin /var/log` (from anywhere) should do the trick? Alternatively if your working directory is `/var`, then `tspin log/` should do the same.

**Reply by bensadeh** [reply, marked as answer]
Watching folders has been removed as of `5.0.0`.
